pyeudiw/trust/__init__.py

```diff
@@ -127,9 +140,12 @@ def _handle_x509_pem(self):
                 "a recognizable Trust Anchor."
             )
 
-        pem = trust_anchor['x509']['pem']
-
-        _is_valid = verify_x509_anchor(pem)
+        try:
+            pem = trust_anchor['x509']['pem']
+            _is_valid = verify_x509_anchor(pem)
+        except KeyError:
```

Comment:
what if the exception is raised by verify_x509_anchor?
except KeyError is not sufficient

Reply:
Resolved in [fix: fixed x509 verification exception handling](https://github.com/italia/eudi-wallet-it-python/pull/188/commits/bf3843cf5708b8826861ae968add41388d6fe0e9)